OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs

```diff
@@ -164,10 +164,7 @@ void SyncMultiplayerCount()
 				var index = int.Parse(name.Substring(5));
 
 				if (index >= newCount)
-				{
 					Players.Players.Remove(name);
-					worldRenderer.World.Players.RemoveAll(pp => pp.InternalName == name);
```

Comment:
I assume this is a workaround @pchote forgot to remove when rewriting the editor with his editor world concept.